src/Nav/Nav.tsx

```diff
@@ -5,7 +5,7 @@ interface INavProps {
 }
 
 export const Nav = styled.nav<INavProps>`
-  z-index: 11;
+  z-index: 2;
```

Comment:
Does this fuck up any other relative indices

Reply:
Oh yeah it does. We have a bunch set to `10`. I'll fix that.